ui/src/services/kolay/docs.ts

```diff
@@ -81,6 +87,33 @@ export default class DocsService extends Service {
      * These can be used to add features syntax-highlighting to pre elements, etc
      */
     rehypePlugins?: UnifiedPlugin[];
+
+    /**
+     * Configuration for what to do when certain events happen.
+     */
+    on?: {
```

Comment:
these events may still be valuable though!

but they don't have to be implemented via the routing system